REPL crash with Eql

I have to admit the reporter was pretty stubborn at misreading the documentation in order to get there, and then trying once more, but anyways...

```Scala
scala> class T extends Eql
1 |class T extends Eql
  |      ^
  |      Cannot extend sealed trait Eql in a different source file
1 |class T extends Eql
  |                ^^^
  |                Missing type parameter for Eql

scala> class T extends Eql
Exception in thread "main" java.lang.AssertionError: assertion failed: denotation module class internal invalid in run 2. ValidFor: Period(1..4, run = 3)
	at dotty.DottyPredef$.assertFail(DottyPredef.scala:17)
	at dotty.tools.dotc.core.Denotations$SingleDenotation.updateValidity(Denotations.scala:828)
```

--- a/compiler/src/dotty/tools/dotc/core/SymDenotations.scala
+++ b/compiler/src/dotty/tools/dotc/core/SymDenotations.scala
@@ -117,7 +117,7 @@ trait SymDenotations { this: Context =>
   /** Possibly accept stale symbol with warning if in IDE */
   def acceptStale(denot: SingleDenotation): Boolean =
     staleOK && {
-      ctx.echo(denot.staleSymbolMsg)
+      ctx.debugwarn(denot.staleSymbolMsg)
       true
     }
 }

--- a/compiler/test-resources/repl/i7644
+++ b/compiler/test-resources/repl/i7644
@@ -1,0 +1,14 @@
+scala> class T extends Eql
+1 | class T extends Eql
+  |       ^
+  |       Cannot extend sealed trait Eql in a different source file
+1 | class T extends Eql
+  |                 ^^^
+  |                 Missing type parameter for Eql
+scala> class T extends Eql
+1 | class T extends Eql
+  |       ^
+  |       Cannot extend sealed trait Eql in a different source file
+1 | class T extends Eql
+  |                 ^^^
+  |                 Missing type parameter for Eql

